public/js/studentController.js

```diff
@@ -1,20 +1,41 @@
-(function () {
-    "use strict"
+'use strict';
+
+(() => {
 
     function getStudentById(id) {
         $.ajax({
-            method: "GET",
-            url: "/api/students/" + id
-        })
-        .done(student => {
-            console.log(student)
-        });
+            method: 'GET',
+            url: '/students?mat=' + id
```

Comment:
Why not to use REST API pattern "/api/students/2017001"? 
Look at "Provide Sensible Resource Names" into [this tips](http://www.restapitutorial.com/lessons/restquicktips.html)